File: DokanyAPI/DokanyApp.BLL/Entities/Product.cs
using System;
using System.Collections.Generic;

namespace DokanyApp.BLL
{
    public partial class Product
    {
        public Product()
        {
            CartItem = new HashSet<CartItem>();
        }

        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public string BrandName { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public string ImageUrl { get; set; }
        public int CategoryId { get; set; }
        public int Quantity { get; set; }
        public string CreationDate { get; set; }
        public ProductAppreciateENU ProductAppreciate { get; set; }

        public virtual Category Category { get; set; }
        public virtual OrderDetails OrderDetails { get; set; }
        public virtual ICollection<CartItem> CartItem { get; set; }
    }
}

File: Dokany UI/src/app/admin/models/Product.ts

export class Product {
    Id: Number;
    Name: String;
    BrandName: String;
    Quantity: Number;
    Price: Number;
    Category: String;
    Description: String;
    CreationDate: Date;
    ImagePaths: String[];

    constructor(_id: Number,
        _name: String,
        _brandName: String,
        _qantity: Number,
        _price: Number,
        _category: String,
        _description: String,
        _creationDate: Date,
        _imagePaths: String[]) {
        this.Id = _id;
        this.Name = _name;
        this.BrandName = _brandName;
        this.Quantity = _qantity;
        this.Price = _price;
        this.Category = _category;
        this.Description = _description;
        this.CreationDate = _creationDate;
        this.ImagePaths = _imagePaths
    }
}